sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeHealthcareEntityAsyncClient.java

```diff
@@ -79,14 +88,51 @@ class AnalyzeHealthcareEntityAsyncClient {
             final Context finalContext = getNotNullContext(context)
                                              .addData(AZ_TRACING_NAMESPACE_KEY, COGNITIVE_TRACING_NAMESPACE_VALUE);
             final boolean finalIncludeStatistics = options.isIncludeStatistics();
+            final StringIndexType finalStringIndexType = StringIndexType.UTF16CODE_UNIT;
+            final String finalModelVersion = options.getModelVersion();
+            final boolean finalLoggingOptOut = options.isServiceLogsDisabled();
+
+            if (service != null) {
+                return new PollerFlux<>(
+                    DEFAULT_POLL_INTERVAL,
+                    activationOperation(
+                        service.submitJobWithResponseAsync(
+                            new AnalyzeTextJobsInput()
+                                .setAnalysisInput(
+                                    new MultiLanguageAnalysisInput().setDocuments(toMultiLanguageInput(documents)))
+                                .setTasks(Arrays.asList(
+                                    new HealthcareLROTask().setParameters(
+                                        (HealthcareTaskParameters) new HealthcareTaskParameters()
+                                                                       .setStringIndexType(finalStringIndexType)
```

Comment:
fhirVersion?

Reply:
FHIR version will be address in a separate PR